### ISSUE TITLE
A11y_VSCode_ ProblemsPanel_ScreenReader: NVDA announcing the “Filter using flies exclude setting” as check box.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#77175]
**Environment Details:**
VSCode Version : 1.360(user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1.Open Visual studio code editor.
2.Using "ctrl+shift+M" key open problem panel.
3.Start screen reader
4.Navigate to "Filter Search" option using keyboard 

**Actual:**
NVDA reads the “Filter using flies exclude setting” as check box

This issue has been observed in "Win 10 + JAWS" and "Win10 + NVDA".

**Expected:**
Screen reader should announce the “Filter using flies exclude setting” as “Search button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader user can get confused about the search button.

**MAS Reference:**
MAS 1.3.1 - 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
[VSCode_ ProblemsPanel_ScreenReader NVDA announcing the “Filter using flies exclude setting” as check box.pptx](https://github.com/microsoft/vscode/files/3380548/VSCode_.ProblemsPanel_ScreenReader.NVDA.announcing.the.Filter.using.flies.exclude.setting.as.check.box.pptx)

**Comment by isidorn** [comment]
The filter using files is a checkbox.
It has 2 states, turned on and off.
Reading it as a checkbox is completly valid. We just visualisy it not as a native checbox but as a styled vscode checkbox.

Thus closing this as invalid
